### ISSUE TITLE
Output event arguments on assertion failure

### ISSUE THREAD

**Issue by zulhfreelancer** [rkalis/truffle-assertions#2]
Hi,

First of all, I would like to say thank you for this awesome and useful library. I'm currently using it in my work. 🙂

Just curious, why the `indexed` event parameter is not showing up when test error is happening?

This is what I saw:

```
Events emitted during test:
---------------------------
LogDeleteUser(userAddress: <indexed>, index: 2)
LogUpdateUser(userAddress: <indexed>, index: 2, userEmail: alice@example.com, userAge: 32)
---------------------------
```

Maybe a bug?

**Comment by rkalis** [comment]
Hi @zulhfreelancer,

Thanks for your message and thanks for opening this issue 🙂.

If I understand you correctly, your issue is that the log shows `<indexed>` instead of the actual address.

However, this logging of the emitted events on test failure is standard Truffle functionality, not part of this package. I tried looking for the reason it does this, and I found [a user with the same question](https://stackoverflow.com/questions/49966223/why-is-truffle-returning-indexed-instead-of-the-actual-value-of-the-event?rq=1), but I found no real answers as to why this is happening.

For what it's worth, the truffle-assertions package should be able to handle assertions with indexed parameters just like regular parameters.

Let me know if I understood you correctly and if this answered your question.

**Comment by zulhfreelancer** [comment]
Yes @rkalis, you get it right. Would be great if truffle-assertions can help users by handling this issue and showing the `<indexed>` values. 🙂

**Comment by rkalis** [comment]
Hmm, makes sense. I'll look at improving error logging. Thanks for the suggestion.

**Comment by zulhfreelancer** [comment]
Thank you @rkalis. 🙂

**Comment by rkalis** [comment]
Hi @zulhfreelancer, 

I added the functionality to show all emitted events in the assertion error message whenever an assertion fails. I also added a prettyPrintEmittedEvents function that shows all events that were emitted in a certain transaction. The README is updated to show this functionality, and I manually tested it with indexed parameters, which also worked. Good luck, and thanks again for the suggestion.

The new functionality is included from 0.3.0 onwards.
